### ISSUE TITLE
Serving assets from CDN

### ISSUE THREAD

**Issue by rymohr** [vigetlabs/gulp-rails-pipeline#9]
Is it possible to precompile assets and serve them from a CDN instead?  I've been using [asset_sync](https://github.com/rumblelabs/asset_sync) with the default rails pipeline and it's been working great.

**Comment by greypants** [comment]
Yeah, definitely possible. I haven't done it yet personally, but here are some resources to get you started.

This option in `rev-collector` https://www.npmjs.com/package/gulp-rev-collector#dirreplacements should let you get your cdn path in there. There are other gulp plugins for uploading. Here are a couple for publishing to amazon web services:

https://www.npmjs.com/package/gulp-s3-upload
https://www.npmjs.com/package/gulp-awspublish

**Comment by rymohr** [comment]
As is the project completely ignores `asset_host` though right?

**Comment by greypants** [comment]
Yeah, none of the tasks know about Ruby or Rails.

**Comment by joaovpmamede** [comment]
@rymohr Did you find a way of getting the assets from a CDN?
From what I get rev-collector only changes files from html templates but we probably need to get other assets referenced on css also.

**Comment by rymohr** [comment]
@joaovpmamede I haven't actually tried to use this one yet.  The project I was hoping to use it for is a production app with 50k lines of js so it won't be an easy migration.  Waiting till it's been battle tested a bit more.
